demisto_sdk/commands/common/constants.py

```diff
@@ -350,7 +351,7 @@
 
 PLAYBOOK_REGEX = r'{}(?!Test){}/playbook-.*\.yml$'.format(CAN_START_WITH_DOT_SLASH, PLAYBOOKS_DIR)
 
-TEST_PLAYBOOK_REGEX = r'{}{}/playbook-.*\.yml$'.format(CAN_START_WITH_DOT_SLASH, TEST_PLAYBOOKS_DIR)
+TEST_PLAYBOOK_REGEX = r'{}{}/(?!script-).*\.yml$'.format(CAN_START_WITH_DOT_SLASH, TEST_PLAYBOOKS_DIR)
```

Comment:
we can delete this one once the TestPlaybooks folder is fully cleared...

Reply:
What do you mean? Is there any change planned?

Reply:
this one is for TestPlaybook folder in the root of content (not inside packs)